src/core.c/CompUnit/Repository/Installation.rakumod

```diff
@@ -18,13 +19,13 @@ class CompUnit::Repository::Installation does CompUnit::Repository::Locally does
     CompUnit::RepositoryRegistry.run-script("#name#");
 }';
 
-
-    method TWEAK() {
+    method TWEAK(:$wrapper-mode) {
         $!lock       := Lock.new;
         $!loaded     := nqp::hash;
         $!seen       := nqp::hash;
         $!dist-metas := nqp::hash;
         $!precomp-store := $!precomp-stores := nqp::null;
+        self!config(:$wrapper-mode) with $wrapper-mode;
```

Comment:
Won't this open a file and potentially parse json even when no modules are loaded? I think we would want to avoid that since we have e.g. `LazyMetaReader` which is mostly convoluted logic around not having to parse json unnecessarily.

Reply:
I'll have a look.

Reply:
I've made it lazy as well.